README.md

```diff
@@ -43,9 +43,22 @@ rokka_cli:
 The commands `organization:create` and `user:create` have a `--save-as-default` option to create and save
 the new values to the `rokka.yml` defaults file. 
 
-## Building rokka-cli.phar
+## Installation: Composer
+
+ - `composer require rokka/client-cli`
+ - run the CLI from `vendor/bin/rokka-cli`
```

Comment:
imho we should not even propose this mode of installation. it is confusing to offer two ways that lead to the bin being a different folder.